server/internal/usecase/repo/schema.go

```diff
@@ -11,6 +11,7 @@ type Schema interface {
 	Filtered(filter WorkspaceFilter) Schema
 	FindByIDs(context.Context, id.SchemaIDList) (schema.List, error)
 	FindByID(context.Context, id.SchemaID) (*schema.Schema, error)
+	FindFieldByIDs(context.Context, id.FieldIDList) ([]*schema.Field, error)
```

Comment:
```suggestion
```

It is not permitted to take only a portion of the aggregate out of the repository. Recall the definition of an aggregate.

More to the point, looking up a schema by field ID is very inefficient; it increases the DB indexes. There is an easier way. When you create some reference field for a schema, you can make it have the ID of the schema it refers to.

Reply:
Just to clerify any misunderstanding, I am getting fields by fieldIds.
I am using this for the corresponding field resolver
```
  SchemaFieldReference:
    fields:
      correspondingField:
        resolver: true
```
Are you suggesting that I should remove the resolver and instead I should save the schema Id??